plugins/kubernetes/app/models/kubernetes/resource.rb

```diff
@@ -348,7 +348,8 @@ def request_delete
           update
```

Comment:
what's the chance of two successive calls returning the same number?

Reply:
what numbers are you talking about ?